### ISSUE TITLE
Policy loss in sac

### ISSUE THREAD

**Issue by sandipan1** [araffin/robotics-rl-srl#31]
In sac.py line 405 , Can you please explain the policy loss term
 loss_policy = (log_pi * (log_pi - q_value_new_actions + value_pred).detach()).mean() 

According to the SAC paper equation 12 it seems to me , it should be 
loss_policy = (log_pi - q_value_new_actions).mean()

**Comment by araffin** [comment]
Hello,
Thank you for your remark. From what I remember, I took the loss term from the original implementation. In the paper, I think they provide only the gradient form.
Yet, I have the same doubt as you. I think opening an issue on the original sac repo would be the solution.
I'll check paper+ implementation this weekend ;)

**Comment by araffin** [comment]
@sandipan1 It seems that they recently updated their code and paper (e.g. they are now using two q-values estimators as in td3). I plan to update our codebase too ;)

**Comment by araffin** [comment]
@sandipan1 as the SAC branch will be merged soon in stable baselines, this should be integrated soon in the toolbox ;)

See https://github.com/hill-a/stable-baselines/pull/120
